### DISCUSSION TITLE
List npm packages by size?

### DISCUSSION THREAD

**Discussion by redbrain** [npm/feedback#193] [General]
Some projects like [npmpopularity.com](https://npmpopularity.com/) can display the most popular packages in the registry.
I was wondering if it was possible to retrieve the largest/smallest packages in the registry, or sort by uncompressed size.

**Comment by ljharb** [comment]
I think that would be really harmful information to expose, as the size of a package is almost never the metric a consumer needs to consider when comparing solutions (since there's almost never two otherwise identically correct and well-maintained packages).

**Comment by MylesBorins** [comment]
There is already an open discussion regarding different views of sorting data, might be good to add to that discussion rather than starting a new one.

https://github.com/npm/feedback/discussions/72

**Reply by redbrain** [reply]
Thank you for letting me know; I didn't see this when I looked through the discussion boards initially.
